cmd/influxd/downgrade/downgrade.go

```diff
@@ -79,7 +90,13 @@ The target version of the downgrade must be specified, i.e. "influxd downgrade 2
 			Flag:    "bolt-path",
 			Default: filepath.Join(v2dir, bolt.DefaultFilename),
 			Desc:    "path for boltdb database",
-			Short:   'm',
+			Short:   'b',
```

Comment:
`m` is the short form used for `bolt-path` in `influxd upgrade`. I thought it was also used in the main `influxd` but I seem to have imagined that... my preference would be to drop the `Short` forms of both this and `sqlite-path`

Reply:
Gotcha. Makes sense to drop the short forms here so I went ahead and did that.